packages/cli/generators/openapi/schema-helper.js

```diff
@@ -160,7 +160,13 @@ function mapObjectType(schema, options) {
       );
       // The property name might have chars such as `-`
       const propName = escapePropertyOrMethodName(p);
+      
       let propDecoration = `@property({name: '${p}'})`;
+   
+      if (required.includes(p)) {
+		propDecoration = `@property({name: '${p}', required: true})`;
```

Comment:
Please run `npm run lint:fix` to fix style issues.

Reply:
done